clientv1_users.go

```diff
@@ -25,6 +25,8 @@ func (s *UsersServiceV1) newClient(ctx context.Context) clientsv1connect.UsersSe
 
 // GetUserByID returns the SAMS user with the given ID. It returns ErrNotFound
 // if no such user exists.
+//
+// Required scope: profile
 func (s *UsersServiceV1) GetUserByID(ctx context.Context, id string) (*clientsv1.User, error) {
```

Comment:
Should this be on the RPC docs instead? i.e. in the proto `service SessionsService`

Reply:
I think that will propagate it to clients and the generated interface docstrings, e.g. https://sourcegraph.sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/internal/telemetrygateway/v1/telemetrygateway_grpc.pb.go?L49-55 is in sync with https://sourcegraph.sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/internal/telemetrygateway/v1/telemetrygateway.proto?L19-24

Reply:
We can add required scope also to the proto definition but users of SDK won't see them (without digging up the source code), they're only seeing the docstring of the SDK methods (like what's added here) because generated methods are almost always gets wrapped (100% wrapped at the moment).